server/settings/settings.go

```diff
@@ -99,6 +99,8 @@ func (s *Server) Get(ctx context.Context, q *settingspkg.SettingsQuery) (*settin
 		UserLoginsDisabled: userLoginsDisabled,
 		KustomizeVersions:  kustomizeVersions,
 		UiCssURL:           argoCDSettings.UiCssURL,
+		UiBannerContent:    argoCDSettings.UiBannerContent,
```

Comment:
I'm pretty sure the banner content and URL should be shown only to authenticated users. Please move `argoCDSettings.UiBannerContent` and `argoCDSettings.UiBannerURL` under `f sessionmgr.LoggedIn(ctx) || s.disableAuth` few lines below